### ISSUE TITLE
Launch by Command line

### ISSUE THREAD

**Issue by JJWatMyself** [seleuco/MAME4droid-2024#71]
I would like to be able to launch a game but command line.  Is this possible and if yes, would it be a tweaked version of the following?

`am start -n com.seleuco.mame4d2024/com.seleuco.mame4d2024.MAME4d2024 /storage/0123-4567/Games/MAME/roms/mame0272-full/puckmap.zip`

**Comment by seleuco** [comment]
it is only possible through an android mechanism called intent. there are android frontends like ES-DE that send CLI commands to mame4droid this way but it has to be done programmatically.

**Comment by JJWatMyself** [comment]
You are absolutely correct.  :) Sorry, I should have been more specific. I'm testing with Beacon (front end). It's not aware of mame4d2024 and suggesting I provide the launch command. I'll test this out some more. If/when that part is working I'll submit a feature on Beacon to eliminate the need to manually configure.

**Comment by seleuco** [comment]
Here you have a lot of information on how to integrate front ends

https://github.com/seleuco/MAME4droid-2024/issues/24
